public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+
+Quite simply, the hack, as described in Ed Nather's account, is impossible on the RPC-4000. The opcode `(C)` field, supposedly modified by the overflow, is in the least significant bits of the instruction. In the terms used above:
+
+             XNNNAAACCC
+        MSB <----------> LSB
+
+
+Thus, any overflow (which progresses toward the MSB) in the bits above the opcode, would not affect the latter. Furthermore, opcode `0` was not "A Jump instruction", but rather one of two operations - `HLT` or `SNS` - the specifics of which are beyond the scope of this analysis. Thus, even a different bit arrangement would not have redeemed the described hack.
+
+
+
+### Reconstructing the Hack...
+
+Obviously, once we rule out Nather's code flow, all options are on the table, including the possibility that the whole thing is made up. However, it's interesting to speculate about scenarios that resemble the one described in The Story of Mel. Further browsing through the sources that Tomer had collected, revealed that the discrepancy between the story and the machine specs did not escape other Mel enthusiasts. David Nugent's [Excellent Writeup](https://www.freecodecamp.org/news/macho-programmers-drum-memory-and-a-forensic-analysis-of-1960s-machine-code-6c5da6a40244/) discusses the problem, but still suggests a flow with the impossible opcode overflow. The [YCombinator discussion](https://news.ycombinator.com/item?id=20489273) contains an excellent analysis of the problem by "YeGoblynQueenne" \([Stassa Patsantzis](https://github.com/stassa)\), including a brief outline of a mechanism described below.
+
+### A Pure Overflow scenario
+
+It turns out that the architecture of the RPC-4000 does provide for a code layout which would accomplish the feat by using an overflow. Using our simplified bit layout, let's assume that the instruction, at some point, reaches the value:
+
+            0111111CCC
+        MSB <----------> LSB
+
+In this diagram, the opcode doesn't matter, it can be any part of the program logic. The address of the next instruction is `111`, so that's where the next step of the loop is located. The data address is also `111`, which doesn't pose a problem: The instruction may not even need an operand, or the value in the `111` address may be commensurate with the program logic. Normally, the program would proceed to the instruction in location `111`. Now, when we try to increment the data address by `1` (adding `1000`), the "overflow" of the field zeroes out the `(A)` and `(N)` fields, yielding this instruction:
+
+            1000000CCC
+        MSB <----------> LSB
+
+
+Which would execute opcode `CCC` and then jump to address 0, just as Ed Nather wrote. In the above diagram, the index register bit is set to `0`, so that it would toggle to `1` following the address overflow. This toggle may be the origin of Nather's recollection of seeing the bit turned on for no apparent reason.
+
+
+### ...A Less Romantic Alternative
```

Comment:
Remove ellipsis

Reply:
Please verify and resolve

Reply:
Consider separating this long paragraph (which is the actual message of the article) into few slices. Either by adding H3s, or separating it to more H2 sections.